scripts/superdesk-authoring/versioning/versions/views/versions.html

```diff
@@ -1,16 +1,6 @@
 <div class="article-versions" ng-controller="VersioningWidgetCtrl">
     <ul class="versions-list card-list">
-        <li class="item draft" ng-class="{selected: selected === item._autosave}" ng-if="item._autosave" ng-click="openAutosave()">
```

Comment:
Issue here is if the autosave is in progress and somebody has version widget open and click on the previous version then you loose the latest changes from autosave version.  Hence on the closing the version widget you don't get the autosave changes.

Made changes such that version history won't show the autosave changes. You have indication that save button is enabled and you cannot preview previous version. You need to save current version to start previewing the all the previous versions.